siti_tools/siti.py

```diff
@@ -482,19 +482,24 @@ def eotf_hlg(
         Returns:
             frame_data: pixel values in the physical luminance up to 10,000 cd/m^2
         """
-        frame_data = np.maximum(frame_data, 0.0)
```

Comment:
According to the [ITU-R Recommendation BT.2100](https://www.itu.int/dms_pubrec/itu-r/rec/bt/R-REC-BT.2100-1-201706-S!!PDF-E.pdf), the values below 0 and above 1 are also possible and acceptable (see NOTE 5g).